test/index.js

```diff
@@ -764,6 +777,11 @@ test('traverse', function() {
   eq(Z.traverse(Identity, identInc, [1, 2, 3]), Identity([2, 3, 4]));
   eq(Z.traverse(Identity, identInc, Nil), Identity(Nil));
   eq(Z.traverse(Identity, identInc, Cons(1, Cons(2, Cons(3, Nil)))), Identity(Cons(2, Cons(3, Cons(4, Nil)))));
+
+  assert.doesNotThrow(function() {
+    // traverse and run the accumulator. this should stack safe.
+    Z.traverse(StateL[FL.of], subAccum, range(70000))._f(100);
```

Comment:
@safareli not easy to check it! the results is an object with key accum and value, the value one contains array length ```70000```, and the member subtracted (100) from the left, so it
```
[0 - 100, 1 - (0 - 100), 2 - (1 - (0 -100)), 3 - 2 - (1 - (0 -100)), ... ]
```
if you know how to tests it, i will happy to add it.